docs/0012_auth_allow_anonymous_sign_ins.md

```diff
@@ -0,0 +1,31 @@
+Level: INFO
+
+### Rationale
+
+Anonymous users use the same `authenticated` Postgres role as permanent users when accessing the database. If you have enabled anonymous sign-in for your project, existing RLS policies may allow unintended access to an anonymous user's JWT.
```

Comment:
> existing RLS policies may allow unintended access to an anonymous user's JWT.

This has no meaning -- what do you mean access to an anonymous user's JWT?

An anonymous user is a user without an email address. It still has a real user ID. 

If a user with an email address can do shady stuff, so can an anonymous user. By turning on anonymous sign-ins you are **not decreasing** the security of your system.

Reply:
it's unintended if it's unexpected - i think it's still worth a reminder to call out policies that allow anonymous users access so that a developer can update them if necessary